backend/cmd/cluster_counter/main.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	count "github.com/Avi18971911/Augur/pkg/count/service"
+	"github.com/Avi18971911/Augur/pkg/elasticsearch/bootstrapper"
+	"github.com/Avi18971911/Augur/pkg/elasticsearch/client"
+	logModel "github.com/Avi18971911/Augur/pkg/log/model"
+	logService "github.com/Avi18971911/Augur/pkg/log/service"
+	spanModel "github.com/Avi18971911/Augur/pkg/trace/model"
+	spanService "github.com/Avi18971911/Augur/pkg/trace/service"
+	"github.com/elastic/go-elasticsearch/v8"
+	"go.uber.org/zap"
+	_ "google.golang.org/grpc/encoding/gzip"
+	"time"
+)
+
+func getAllDocumentsQuery() map[string]interface{} {
+	return map[string]interface{}{
+		"query": map[string]interface{}{
+			"match_all": map[string]interface{}{},
+		},
+	}
+}
+
+func deleteAllDocuments(es *elasticsearch.Client) error {
+	indexes := []string{bootstrapper.CountIndexName}
+
+	query := map[string]interface{}{
+		"query": map[string]interface{}{
+			"match_all": map[string]interface{}{},
+		},
+	}
+	queryJSON, _ := json.Marshal(query)
+	res, err := es.DeleteByQuery(indexes, bytes.NewReader(queryJSON), es.DeleteByQuery.WithRefresh(true))
+	if err != nil {
+		return fmt.Errorf("failed to delete documents by query: %w", err)
+	}
+	defer res.Body.Close()
+
+	if res.IsError() {
+		return fmt.Errorf("failed to delete documents in index %s", res.String())
+	}
+	return nil
+}
+
+func main() {
```

Comment:
Made this main to test out the counting separately from clustering + insertion of data. This paves the way for better data pipelining.

Reply:
It also has no errors! Took me such a long time to avoid conflicts in the DB. I pray that this is true, and not the errors being masked or operations not actually being executed.